keeper-gradle-plugin/src/main/kotlin/com/slack/keeper/VariantClasspathJar.kt

```diff
@@ -194,5 +198,27 @@ abstract class AndroidTestVariantClasspathJar : BaseKeeperJarTask() {
     diagnostic("${archiveFile.get().asFile.nameWithoutExtension}Classes") {
       androidTestClasses.sorted().joinToString("\n")
     }
+
+    // See https://issuetracker.google.com/issues/157583077 for why we do this
+    if (emitDebugInfo.get()) {
+      val duplicateClasses: Set<String> = appJars.asSequence()
```

Comment:
This is checking the "the same class is present in both the app APK and test APK" condition but not the "they come from different JARs" condition that you identify in the bug report, right? (Just want to make sure I'm following along here.)

Reply:
This is checking classes only from the jars unique to the androidTest APK, so it's implicitly different jars